src/java/org/apache/cassandra/cql3/statements/schema/TableAttributes.java

```diff
@@ -84,6 +91,24 @@ public TableId getId() throws ConfigurationException
         }
     }
 
+    /**
+     * Returs `true` if this attributes instance has a COMPACTION option with a recognized unsupported compaction
+     * strategy class (coming from DSE). `false` otherwise.
+     */
+    boolean hasUnsupportedDseCompaction()
+    {
+        if (hasOption(Option.COMPACTION))
+        {
+            Map<String, String> compactionOptions = getMap(Option.COMPACTION);
```

Comment:
are we positive that the strategy names are case sensitive?
I.e. e.g. was `MeMoryOnlyStrategy` ever a valid strategy?
If not -we're good. If yes - we need to adjust.

Reply:
`MeMoryOnlyStrategy` is not a valid strategy, the names are case sensitive.